main/options/options.cc

```diff
@@ -837,6 +841,21 @@ void readOptions(Options &opts,
         }
         opts.stripeMode = raw["stripe-mode"].as<bool>();
         opts.stripePackages = raw["stripe-packages"].as<bool>();
+        if (raw.count("extra-package-files-directory-prefix")) {
+            if (!opts.stripePackages) {
+                logger->error("--extra-package-files-directory-prefix can only be specified in --stripe-packages mode");
+                throw EarlyReturnWithCode(1);
+            }
+            for (const string &dirName : raw["extra-package-files-directory-prefix"].as<vector<string>>()) {
```

Comment:
WDYT about checking that these exist on disk as directories here?

Reply:
That's a good idea -- we can add that at a later time :)